### DISCUSSION TITLE
请问各位老师，当 fallback-filter 下的geoip：false 时，fallback是不是只有在nameserver不响应才调用

### DISCUSSION THREAD

**Discussion by wzy105** [vernesong/OpenClash#3746] [Q&A]
配置:
    fallback-filter: { geoip: false}

这时fallback是不是只有在nameserver没响应才会调用。

**Comment by ghost** [comment]
不是的，fallback是和nameserver并发的，fallback filter只是控制用不用来自fallback的dns解析结果的

你在找的可能是nameserver policy

**Reply by ghost** [reply]
https://wiki.metacubex.one/config/dns/diagram/
看最下面的流程图
